packages/fuel-indexer-lib/src/utils.rs

```diff
@@ -229,71 +236,3 @@ pub async fn init_logging(config: &IndexerConfig) -> anyhow::Result<()> {
     }
     Ok(())
 }
-
-pub fn format_exec_msg(exec_name: &str, path: Option<String>) -> String {
```

Comment:
These functions are only used by forc-index so they were moved to forc-index

Reply:
Nice. 👌🏽